lib/pages/connect/connect_page_mixin.dart

```diff
@@ -74,6 +84,30 @@ mixin ConnectPageMixin {
     );
   }
 
+  Future<void> tryLogoutSso(BuildContext context) async {
+    final redirectUrl = _generateRedirectUrl();
+    final url = _getLogoutUrl(context, redirectUrl: redirectUrl);
+    if (url == null) return Future.value();
+
+    final urlScheme = _getRedirectUrlScheme(redirectUrl);
+    if (PlatformInfos.isWeb) {
+      FlutterWebAuth.authenticate(
+        url: url,
+        callbackUrlScheme: urlScheme,
+      );
+    } else {
+      try {
+        final result = await FlutterWebAuth.authenticate(
+          url: url,
+          callbackUrlScheme: urlScheme,
+        );
+        Logs().d('tryLogoutSso::result: $result');
+      } catch (e) {
```

Comment:
What is the different?

Reply:
fixed. web no need separate logic anymore